compile.py

```diff
@@ -0,0 +1,78 @@
+import yaml
+
+rows = []
+with open('cfp.yml', 'r') as stream:
+    yaml_content = yaml.safe_load(stream)
+    rows.append(dict(yaml_content))
+
+header = '''# Curated List of CFPs
```

Comment:
@uchitsa you don't need `header` and `bottom` in this file. We already have in the `README.md`. You just load `README.md`, find the place where to insert new markdown piece, and then save the file.

Reply:
@yegor256 exactly, thanks! updated